development/mob_programming.md

```diff
@@ -0,0 +1,45 @@
+# Mob Programming for Research
+
+## "Mob programming"?
+
+[mobprogramming.org](https://mobprogramming.org/) describes it as "All the brilliant people working on the same thing, at the same time, in the same space, and on the same computer". These words are from Woody Zuill, one of the folks behind the original Agile Manifesto, who developed the practise as a way of recovering the _intention_ of agile software creation in a world of Scrum Coach certifications and process tooling. If you haven't done pair programming, then mob programming is more fun and easier (after a while).
+
+If you do it formally, there are rules:
+
+* One person is the _driver_, and they don't make any decisions about the code or configuration that's being written
+* Another may be nominated as the _navigator_, primarily making the suggestions, or feedback may come from the whole group
+* The driver rotates in quite short periods, 5 to 10 minutes
+
+## Helping it work for research
+
+The formal rules don't always work well in a research setting where there are a lot of speculative decisions to be made, different paths to go a short way down, and often complex dependencies. Here are some rules of thumb developed in different settings of mob programming with a mixed group of software engineers and researchers.
+
+* Keep the group size to no more than 4, ideally 3 - larger than this, and people tend to drift off.
+* Try to rotate the screenshare at least every 20 minutes to help keep everyone engaged, though there may be some settings where it makes sense for the least experienced person to "drive" the whole way through
+* When starting a new project or onboarding a new person, allow at least a whole session for environment setup
```

Comment:
Could you move this point?   You mention session before you define it, might better at the the end. Putting it at the end would perhaps allow it to flow better: steps needed, optional, things to remember.